### ISSUE TITLE
PropertyChangedMode Property

### ISSUE THREAD

**Issue by Chicagoan2016** [MarimerLLC/cslaforum#413]
In a Windows Forms applications, is it still recommended to set PropertyChangedMode to Windows?
If yes, where should we set this property?

kind regards

**Comment by rockfordlhotka** [comment]
Yes, if you don't set this then data binding will not refresh properly in some cases.

You can set it once as the app starts up, so in the app main method, or the first window's loaded event handler (assuming you have a single first window that loads).

**Comment by tfreitasleal** [comment]
Or you can set in the `app.config` file, in the `appSettings` section like this
```
<configuration>
  <appSettings>
    <add key="CslaPropertyChangedMode" value="Windows" />
  </appSettings>
</configuration>
```
